orion-ui/src/App.vue

```diff
@@ -129,7 +130,19 @@
     mobileMenuOpen.value = false
   }
 
-  watchEffect(() => document.body.classList.toggle('body-scrolling-disabled', showMenu.value && !media.lg))
+  const healthCheck = async (): Promise<void> => {
+    try {
+      await healthApi.getHealth()
+    } catch (error) {
+      showToast('Network error', 'error', { timeout: false })
+      console.warn(error)
+    }
+  }
+
+  watchEffect(() => {
+    healthCheck()
```

Comment:
Why call this in a watch effect?

Reply:
I haven't thought it through yet, just added it there to see if API returns an error since watchEffect was already there. Would it be better to put it in onMounted?

Reply:
I'm not sure it even needs to be in a hook. Should just be able to call it directly

Reply:
True, I don't think I've seen such a pattern of us calling functions directly in our code

Reply:
I would probably move the health check function into a `api` utility file. Then just import and call it in `App.vue`